pupa/settings.py

```diff
@@ -9,6 +9,10 @@
 
 # scrape settings
 
+ENABLE_KAFKA = os.environ.get('ENABLE_KAFKA', "False").lower() == "true"
+KAFKA_REPORT_TOPIC = 'post-scrape-reports'
```

Comment:
Revised proposal would be "post-scrape-reports/:jurisdiction" and "ocd-data/:jurisdiction" as topics. My gut says that we'd like to avoid pre-combining streams and rather have there be a focus on combining streams if you need data from both. We can do aggregation later via something like Samza. I'd like to keep the topics as small as logically possible. 

Why? I'm not sure if we had occasion to bring it up yet, but the way Kafka deals with data sizes larger than a single machine can handle is via something they call "partitions" on topics. Partitions are, glibly, topics within topics, but confined to one machine (while potentially replicated to multiple machines if need be). Messages produced to a large and active topic are sent to one of the partitions based on partition logic (randomly or based on keys). Right now we are messing with topics that only have a single partition. What worries me is that there is no built in ordering between partitions on a topic. There is no way built in to compare the offsets between one partition and another and know what the ordering of the union of the two partitions are. I think we've been designing around a global ordering on all messages pushed to a topic and I'm not sure we want to reconsider the various parts of the producing to Kafka if we can help it, especially if we go down the "push the previously pushed 500 messages on failure" route. 

I'm still trying to estimate the numbers and impact on the design, but I worry that if we combine the events of all the scrapers into one stream then we might overwhelm a single machine and eventually we might have to partition the combined stream and roll our own global ordering of events between partitions. I'm confident the set up we've designed can work if a topic is confined to a single partition (with replicas) with topics distributed across multiple machines if needed. I also think that the results of the scrape that we have envisioned for a single jurisdiction is likely to fit on a single machine for the forseeable future. What I don't know is if you can take the last estimate for topic sizes and multiple it by ~60 for all of the federal and state level legislative data and still remain within the capability of one machine. 

Without a more pressing argument for why combining the jurisdictions makes sense, I think we should save ourselves potentially reworking a lot of our assumptions/designs by pushing reports/ocd-data to their own jurisdiction streams.